docs/product/components/progress-bars.html

```diff
@@ -231,6 +231,99 @@
     </div>
 </section>
 
+<section class="stacks-section">
+    {% header "h2", "Segmented" %}
+    <p class="stacks-copy">For task tracking, switching to a segmented progress bar might be more appropriate. Our segmented
+        progress bar inherits the parent’s text color. You and your designer will need to choose an appropriate font
+        color. You’ll also need to pass the <code class="stacks-code">--s-progress-value-max</code> CSS variable the total number
+        of segments, and the <code class="stacks-code">--s-progress-value</code> CSS variable the number of completed segments..</p>
+
+<div class="stacks-preview">
+{% highlight html %}
+<div class="s-progress s-progress__segmented" style="--progress-value-max: 8; --progress-value: 4">
```

Comment:
This API is nice and super flexible, but I've got a few issues with this approach down below.